### ISSUE TITLE
No slices information

### ISSUE THREAD

**Issue by nickhendo** [up-banking/api#152]
I've scoured the docs and API responses and can't find any information about slices anywhere. Specifically, when a slice is paid, it would be great if there was even a little bit of information, whether that be on the original sliced transaction or on the paid slices, just with a reference to the original transaction being sliced. Without this link unfortunately, these just look like basic transfers, and have no meta information beyond the category.

**Comment by markbrown4** [comment]
Yes, there's no extra information around payment requests / slices in the API yet.
Can see how it'd be useful, but to set expectations this is lower priority than some of the other core feature requests: Payments, Savers, Covers.
